### PR TITLE
[Balance] Table salt radiation intraction

### PR DESCRIPTION
**Pull request by IrkallaEpsilon**
## About The Pull Request
1/10th of the efficiency of Hyronalin while still able to OD yourself on.

I cannot wait to see doctors injecting salt + hyronalin into people if this goes through.

## Changelog
:cl:
balance: Table salt is now a very mild radioprotectant.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 1]
```diff
@@ -360,6 +360,9 @@
 	scannable = TRUE
 	common = TRUE
 
+/datum/reagent/sodiumchloride/affect_blood(mob/living/carbon/M, alien, effect_multiplier) //reasoning: Table salt is usually iodized. Iodine saturates glands. Glands are often getting screwed over by isotopes of iodine if exposed to radiation. Yes its a preventatitve method IRL but this is gamyfictaion.
+	M.radiation = max(M.radiation - (0.3 * effect_multiplier), 0) //10 times less effective than Hyronalin
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/food-Drinks.dm:364]:
```suggestion
	..()
	M.radiation = max(M.radiation - (0.3 * effect_multiplier), 0) //10 times less effective than Hyronalin
```
?

**Comment by cdb-is-not-good** [comment]
Thumbs up

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 2]
```diff
@@ -360,6 +360,10 @@
 	scannable = TRUE
 	common = TRUE
 
+/datum/reagent/sodiumchloride/affect_blood(mob/living/carbon/M, alien, effect_multiplier) //reasoning: Table salt is usually iodized. Iodine saturates glands. Glands are often getting screwed over by isotopes of iodine if exposed to radiation. Yes its a preventatitve method IRL but this is gamyfictaion.
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/food-Drinks.dm:363]: Why is this in blood?
isnt the point of iodine to collect rads then expel them later?
Blood implies that the blood is using rad-collected salt

Comment (Trilbyspaceclone) [reply code/modules/reagents/reagents/food-Drinks.dm:363]: I was just thinking having it on ingest would fit better ;-;

**Pull request merged**